test/other/metadce/test_metadce_mem_O3_standalone_narg_flto.jssize

```diff
@@ -1 +1 @@
-4475
+4553
```

Comment:
Odd that this increases code size?

Reply:
Interesting yes... maybe I'll split out the internal stuff and land that on its own.